src/Symfony/Bundle/FrameworkBundle/Resources/config/messenger.php

```diff
@@ -218,4 +214,24 @@
                 service('messenger.default_bus'),
             ])
     ;
+
+    if (class_exists(StopWorkerOnSignalsListener::class)) {
+        $services
+            ->set('messenger.listener.stop_worker_signals_listener', StopWorkerOnSignalsListener::class)
+                ->args([
+                    null,
+                    service('logger')->ignoreOnInvalid(),
+                ])
+                ->tag('kernel.event_subscriber')
+        ;
+    } else {
+        // BC layer for symfony/messenger < 5.3
```

Comment:
you mean 6.3 I suppose
but let's bump the min version of messenger instead?

Reply:
Oups 😅 + Good catch. Fixed 👍🏼